README.md

```diff
@@ -14,9 +14,20 @@ host_vars, that will override the vars plugin.
 `cumulus_ntp_servers`: List of ntp servers to use - Default: Cumulus ntp servers
 `cumulus_ntp_interface`: Interface to listen on for ntp - Default eth0
 
+## Default management interface
+
+This role uses the following default configuration for the dedicated management port when that port is `eth0`. Note, some Cumulus supported hardware uses `eth1` for the management interface, it is up to the user to know this and edit the default if needed. The configuration can be found in `/templates/header.j2`. The default will result in the following being put in `/etc/network/interfaces` unless `eth0` is defined elsewhere:  
+
+```
+# The primary network interface
+auto eth0
+iface eth0 inet dhcp
+    mtu 1500
+```
+
 ## Port Policies
 
-It may be desirable to set a default switch-wide port configuration, such as MTU. This is simply yaml to json.
+It may be desirable to set a default switch-wide port configuration, such as MTU. This is simply yaml to json.  Note, setting this example policy will apply the MTU 9216 to _all_ ports on the switch that do not have an MTU explicitly set on the interface config.
 
```

Comment:
```suggestion
NOTE: This role by default sets an MTU of 1500 on the `eth0` interface to protect against eth0 becoming inoperable if a global MTU above 1500 is set on hardware where `eth0` is not capable of going above 1500, ie Dell S3048 or Dell Z9100. If one defines this interface elsewhere via `config.interfaces` one may override the MTU to the desired size or remove it altogether.
```

Reply:
I like this version, @russelltadams hit the commit suggestion button and see magic happen